### ISSUE TITLE
What comes next after workshops?

### ISSUE THREAD

**Issue by khamiltonuk** [codebar/chapter-organiser-meetings#2]
Once codebar is stable what is the next step? 
- Can we host regional 24 pull requests
- Should we host workshops on specific technologies
- Should we host codebar more frequently

**Comment by KimberleyCook** [comment]
Yes to more 24 PRs!!! If you want to discuss how these events run that I'd be more than happy to have a call with you.

Yes, but is there appetite for these specific workshops? We wouldn't want you to run one and no-one show up, maybe do a poll on twitter/ask in your chapter slack? Also, make sure you have enough coaches who can teach that specific topic. We do have another event type in the codebar website where you can add more custom communication, custom urls ETC so either myself or @matyikriszta can show you how these work if this is something you want to happen. 

We would love that, but make sure you have the organisation team behind you to help with this. And enough hosts to support. You can become more frequent and then always change back if it's too much.

Action
- Create a new "workshop coordinator" role, where we can get extra help at workshops from people who want to help but don't want to necessarily want to be organisers.
